koku/koku/koku_test_runner.py

```diff
@@ -46,7 +46,6 @@ class KokuTestRunner(DiscoverRunner):
 
     account = "10001"
     schema = f"acct{account}"
-    settings.HOSTNAME = "koku-worker-10-abcdef"
```

Comment:
Do we know why this was set to begin with?

Reply:
this can be removed.  It was a leftover from one of the worker cache fix iterations.

Reply:
This might be an error in merge conflict resolution. I'll double-check.